exploration/check_return_period_funcs.py

```diff
@@ -0,0 +1,179 @@
+# ---
+# jupyter:
+#   jupytext:
+#     cell_metadata_filter: -all
+#     custom_cell_magics: kql
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.11.2
+#   kernelspec:
+#     display_name: ds-raster-stats
+#     language: python
+#     name: ds-raster-stats
+# ---
+
+# %%
+# %matplotlib inline
+# %load_ext autoreload
+# %autoreload 2
+
+# %%
+from src.utils import cloud_utils
```

Comment:
Would also add 

```
import os
os.chdir('..')
```

So that the `src` module is imported properly

Reply:
Can you explain this to me?
-  Is this required on top of the `setup.cfg`,`pyproject.toml` , `_init_.py` files and `pip install -e.` ?
- or does this replace the need for some combination of the above? 

The modules seem to be importing properly on my machine using the top bullet point

Reply:
I think running `pip install -e .` obviates the need for `os.chdir('..')`. Then you can import the `src` module the same way regardless of what directory you're in, with a notebook or a `.py`. Then I also put `pip install -e .` in whatever GH Action workflow file so it runs the same in there too. I am however not sure what the best practice is, it's just based on our coding guidelines.

Reply:
Oh nice! I'm not very familiar with using `pip install -e`. Looks great then.